test/fixtures/schema/variable-property-fixed-type.json

```diff
@@ -144,7 +210,7 @@
                               "attributes": {
                                 "contentType": "application/schema+json"
                               },
-                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"$ref\": \"#/definitions/y\",\n  \"definitions\": {\n    \"y\": {\n      \"type\": \"array\",\n      \"items\": {\n        \"type\": \"object\",\n        \"properties\": {\n          \"name\": {\n            \"type\": \"string\"\n          }\n        },\n        \"required\": [\n          \"name\"\n        ]\n      }\n    }\n  }\n}"
+                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"allOf\": [\n    {\n      \"$ref\": \"#/definitions/y\"\n    },\n    {\n      \"$ref\": \"#/definitions/z\"\n    }\n  ],\n  \"definitions\": {\n    \"y\": {\n      \"type\": \"array\",\n      \"items\": {\n        \"type\": \"object\",\n        \"properties\": {\n          \"name\": {\n            \"type\": \"string\"\n          }\n        },\n        \"required\": [\n          \"name\"\n        ]\n      }\n    },\n    \"z\": {\n      \"type\": \"array\",\n      \"items\": [\n        {\n          \"type\": \"object\",\n          \"properties\": {\n            \"first\": {\n              \"type\": \"string\"\n            }\n          },\n          \"required\": [\n            \"first\"\n          ]\n        },\n        {\n          \"type\": \"object\",\n          \"properties\": {\n            \"last\": {\n              \"type\": \"string\"\n            }\n          },\n          \"required\": [\n            \"last\"\n          ]\n        }\n      ]\n    }\n  }\n}"
```

Comment:
Ah good point. In that case it should use a separate test file otherwise it will drop test coverage for https://github.com/apiaryio/drafter/pull/411.

I've updated PR to take this into account.